lib/output-styles-preprocessor.js

```diff
@@ -24,8 +25,17 @@ OutputStylesPreprocessor.prototype.toTree = function(inputNode, inputPath, outpu
 
   debug('concatenating module stylesheets: %o', concatOptions);
 
+  var parentOptions = options.registry.app.options || {};
+  var cssModulesOptions = parentOptions.cssModules || {};
+  var postprocessPlugins = cssModulesOptions.plugins && cssModulesOptions.plugins.postprocess;
```

Comment:
I think that's right—IIRC that was renamed to distinguish between ECM plugins and PostCSS plugins